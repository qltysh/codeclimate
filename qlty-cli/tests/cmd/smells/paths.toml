args = ["smells", "--no-duplication", "--no-snippets", "--json", "app"]
bin.name = "qlty"
status.code = 0
