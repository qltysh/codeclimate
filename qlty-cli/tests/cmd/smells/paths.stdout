
[
  {
    "tool": "qlty",
    "driver": "structure",
    "ruleKey": "return-statements",
    "message": "Function with many returns (count = 7): init",
    "level": "LEVEL_MEDIUM",
    "language": "LANGUAGE_JAVASCRIPT",
    "category": "CATEGORY_STRUCTURE",
    "snippet": "function init() {/n  if (true) {/n    return;/n  }/n  return;/n  return;/n  return;/n  return;/n  return;/n  return;/n}",
    "snippetWithContext": "function init() {/n  if (true) {/n    return;/n  }/n  return;/n  return;/n  return;/n  return;/n  return;/n  return;/n}",
    "effortMinutes": 20,
    "value": 7,
    "valueDelta": 1,
    "location": {
      "path": "app/init.js",
      "range": {
        "startLine": 1,
        "startColumn": 1,
        "endLine": 11,
        "endColumn": 2,
        "startByte": 0,
        "endByte": 109
      }
    },
    "mode": "MODE_BLOCK"
  }
]
